### ISSUE TITLE
root application in rtmp

### ISSUE THREAD

**Issue by massoudAfrashteh** [arut/nginx-rtmp-module#910]
I defined application config as this:

```
rtmp {
    server {
        listen 1935;
        application /{
            live on;
        }
    }
}
```
And checked with FFMPEG 
ffmpeg -i Princess.avi -f flv rtmp://localhost/massoud
It won't work :| but when I change application to something like myApp it works. Any idea?

Thanks

**Comment by Somosmultimedia** [comment]
Why do you want the root system to be writeable?
Always is a good practice to use a subdirectory writeable for the application, something like /tmp/live

**Comment by massoudAfrashteh** [comment]
Thanks for answer... I don't want to have an application name in my RTMP address e.g rtmp://localhost/massoud use massoud as streamName just that :)

**Comment by Somosmultimedia** [comment]
I guess you are confusing directories with application name.
As far as I know, is mandatory the application name.
I suggest ingest the video with rtmp and play it with HLS

**Comment by massoudAfrashteh** [comment]
No... I know it's mandatory but I want to delete the application name from the first request and pass it to an another RTMP server with 3xx redirection and put my app name is second RTMP config. I don't need to have an application name in my first RTMP server.

**Comment by Somosmultimedia** [comment]
I understand what you want to do.
Instead using a redirect I recomend trying to use a proxy. Here I found something http://serverfault.com/questions/622141/rtmp-proxy-server

**Comment by massoudAfrashteh** [comment]
Thanks but that's just a pull and not wy answer... I just want to have this format in first RTMP request  rtmp://localhost/streamName without application name... Thanks for this incredible repo anyway

**Comment by Somosmultimedia** [comment]
I don't know if its possible to stream rtmp without an application name, I guess not. I did a search online and didn't find anything. =(

**Comment by Somosmultimedia** [comment]
Maybe modifying the c++ module code and compile it. But that's another story

**Comment by massoudAfrashteh** [comment]
Thanks I thought I have to do this I can do it and push that in this repo. Thanks :)
